x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/translations.ts

```diff
@@ -136,6 +143,7 @@ export const rulesErrorReasonTranslationsMapping = {
   license: ALERT_ERROR_LICENSE_REASON,
   timeout: ALERT_ERROR_TIMEOUT_REASON,
   disabled: ALERT_ERROR_DISABLED_REASON,
+  validate: ALERT_ERROR_VALIDATE_REASON,
```

Comment:
The way the reason type is defined, this code generated an error until I added this new reason.  Quite nice - I knew I'd have to do something in the UX, wasn't sure where, and didn't have to go hunting for it, just run the type checker!